src/recordpage.cpp

```diff
@@ -605,10 +620,12 @@ void RecordPage::onApplicationNameChanged(const QString& filePath)
         setError(tr("Application file is not executable: %1").arg(filePath));
     } else {
         const auto config = applicationConfig(filePath);
-        ui->workingDirectory->setText(config.readEntry("workingDir", QString()));
+        ui->workingDirectory->setText(config.readEntry("workdir", QString()));
```

Comment:
why did you change the key here, this could stay at the old name, no?

Reply:
I changed this one by mistage